test/karma.conf.js

```diff
@@ -5,8 +5,12 @@ const CI = Boolean(process.env.CI);
 
 let build = `material-ui local ${new Date().toISOString()}`;
 
-if (process.env.CIRCLE_BUILD_URL) {
-  build = process.env.CIRCLE_BUILD_URL;
+if (process.env.CIRCLECI) {
+  const buildPrefix =
+    process.env.CIRCLE_PR_NUMBER !== undefined
+      ? `#${process.env.CIRCLE_PR_NUMBER}`
```

Comment:
one detail:

`#` seems to be swallow:

<img width="765" alt="Screenshot 2021-05-16 at 11 30 34" src="https://user-images.githubusercontent.com/3165635/118392460-29c3fa80-b63a-11eb-9f0b-19df0401ec28.png">

Reply:
Odd. Well, just the number works as well I suppose.

Reply:
Maybe PR as a prefix could work too 🤷‍♂️